src/main/java/no/entur/uttu/export/netex/NetexExportContext.java

```diff
@@ -59,6 +59,8 @@ public class NetexExportContext {
 
   public Set<DestinationDisplay> destinationDisplays = new HashSet<>();
 
+  public Set<Ref> serviceLinkRefs = new HashSet();
```

Comment:
Let's use a context object here, `ServiceLinkExportContext` -> from quay and to quay, along with mode.